packages/kotti-table/src/TableColumn.js

```diff
@@ -105,6 +110,7 @@ function createColumn(column = {}) {
 
 	column.id = columnId
 	column.type = COLUMN_TYPE
+	column._deleted = false
```

Comment:
Any explanation would be appreciated.

Reply:
whenever columns are hidden and shown or reorderd the are unmounted
this unmounting causes the column to be removed along with all it's state and then later added again, but at the end out of order since it's not longer there

explicitly setting the order isn't practical and I couldn't think of a more practical idea then to simply mark the column as deleted until it is restored again in the next cycle

the column is eventually removed when the entire table is removed, and you can never have 2 columns with the same prop key so no memory leak should occur 

should I add a comment stating needed for maintaining column state across re-renders